doc/manual/plugins_local/saveimage.rst

```diff
@@ -0,0 +1,29 @@
+.. _local-plugin-saveimage:
+
+SaveImage
+=========
+
+.. image:: figures/saveimage_screenshot.png
```

Comment:
I'm going to have you redo the screenshot, because I think you can put together a more interesting one based on a real workflow that you have.  Let's just handle this and any other final tweaks as a separate, new PR.